spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+  updated_params = {
+    name: 'New Name',
+    email: 'newemail@gmail.com'
+  }
```

Comment:
These aren't needed in all of the tests, so you should only put them in the block thats needed